tests/unit/core/test_tracking.js

```diff
@@ -432,20 +434,22 @@ describe(__filename, () => {
     });
   });
 
-  describe('Hybrid Content Telemetry filterIdentifier()', () => {
-    it('should return content up to the default max length', () => {
-      expect(filterIdentifier('a'.repeat(26))).toHaveLength(20);
-    });
+  describe('HCT identifiers', () => {
+    const telemetryRegex = /^[a-z0-9]{1}[a-z0-9_]+[a-z0-9]{1}$/i;
 
-    it('should return content up to the supplied max length', () => {
-      expect(filterIdentifier('a'.repeat(26), { maxLen: 10 })).toHaveLength(10);
+    Object.values(HCT_METHOD_MAPPING).forEach((idString) => {
```

Comment:
Jest introduced a `.each` method on the `it` and `describe` functions. We tend to use it:

```diff
diff --git a/tests/unit/core/test_tracking.js b/tests/unit/core/test_tracking.js
index 3c785a1a9..68e98d143 100644
--- a/tests/unit/core/test_tracking.js
+++ b/tests/unit/core/test_tracking.js
@@ -437,12 +437,13 @@ describe(__filename, () => {
   describe('HCT identifiers', () => {
     const telemetryRegex = /^[a-z0-9]{1}[a-z0-9_]+[a-z0-9]{1}$/i;
 
-    Object.values(HCT_METHOD_MAPPING).forEach((idString) => {
-      it(`should ensure hct method "${idString}" meets HCT identifier requirements`, () => {
+    it.each(Object.values(HCT_METHOD_MAPPING))(
+      'should ensure hct method "%s" meets HCT identifier requirements',
+      (idString) => {
         expect(idString).toMatch(telemetryRegex);
         expect(idString.length).toBeLessThanOrEqual(20);
-      });
-    });
+      },
+    );
 
     for (const action of telemetryObjects) {
       it(`should ensure hct object (${action}) meets HCT identifier requirements`, () => {
```

The function gets the `idString` as first argument and you use `%s` in the test description.

Reply:
Yep that's how `it.each` works, those are positional placeholders.

Reply:
See https://jestjs.io/docs/en/api#1-testeachtable-name-fn-timeout

Reply:
Hmm I clearly got a bit confused here - I somehow thought you were commenting on my use of `it.each` elsewhere. Happy to move to using it here too. I used it in the doc tests already.

Reply:
My bad! That's all fixed now.